### DISCUSSION TITLE
Desktop download link

### DISCUSSION THREAD

**Discussion by HiesukeKurusu** [spartacus04/jext-reborn#488] [Q&A]
Is it possible for you to post the download link to the desktop app? I am not able to get the file to download in any way unless there is somewhere i didn't click.

**Comment by hax4dazy** [comment]
Yeah, please make the download link for the Desktop app an actual release instead of just a github actions thing. Github actions can and will expire after some time which sucks for new users.

**Reply by spartacus04** [reply]
I've setup a Cron job that re runs the GitHub action every week, that way the artifacts won't expire

**Reply by hax4dazy** [reply]
That works as well lol
